### ISSUE TITLE
www.bild.de is working for some seconds, then blocked (Advertisement)

### ISSUE THREAD

**Issue by Videoso** [uBlockOrigin/uAssets#7753]
### Describe the issue

 

Move to `www.bild.de` and scroll through page or wait some time (max up to 1 minute) it will change to this URL `https://www.bild.de/wa/ll/bild-de/privater-modus-unangemeldet-54578900.bild.html` and lost access to ad-free version.

### Versions

- Browser/version: 84.0.4147.105
- uBlock Origin version: v1.29.0

### Settings

-
Easy List / Easy List DEU in use.
### Notes

Use back button in Browser, show it again after around 1 minute it continue in loop with block / back / block / back.

Thanks you

**Comment by okiehsch** [comment]
Does
`@@||tags.tiqcdn.com/utag/axelspringer/bild/prod/utag.js$script,domain=bild.de`
work?

**Comment by Videoso** [comment]
Yes seems working, thank you so much!

**Comment by Videoso** [comment]
Hi,

the issue is back, sorry, may they change something again or still the same.

**Comment by okiehsch** [comment]
`@@||script.ioam.de/iam.js$script,domain=bild.de`

**Comment by Videoso** [comment]
Confirm, updated link / 2nd link is today working again, will monitor it next few days.

**Comment by spodermenpls** [comment]
@mapx- @okiehsch When I was visiting the article `https://www.bild.de/regional/frankfurt/frankfurt-aktuell/luebcke-mord-neonazi-ernst-hatte-kontakt-zu-npd-bundesvize-heise-73589494.bild.html`, after a few seconds I get redirected to this new dead end site: `https://www.bild.de/adblockwall.html`. All uAssets filters are active and fresh, on uBlock Origin 1.20.6 and FF 82.0.

**Comment by mapx-** [comment]
uBO default setup => no issue on my side.
Be sure keeping no other blocking extension or other strange lists

**Comment by spodermenpls** [comment]
@mapx- Okay, thanks for your quick reply. I took another look at uMatrix (that I also use), where everything was allowed except for a "utag.js" from `tags.tiqcdn.com`, that gets blocked by a default uMatrix setup and usually isn't needed (I happen to have three bild.de tabs open, but only this one triggered the redirect), in this case it is. Greenlighting it did the trick, so no changes needed here. Thanks anyway!

**Comment by spodermenpls** [comment]
@mapx- When I visit
https://www.bild.de/bild-plus/regional/hamburg/hamburg-aktuell/autohaendler-muss-zahlen-kohle-krieg-in-feinsten-kreisen-73809898.bild.html, even with uMatrix being disabled and just stock filter lists, the redirect mentioned above persists. Is there something that can be done from uAssets' perspective, to prevent uBlock from triggering this site's adblock detection?

**Comment by mapx-** [comment]
- Tested again in FF, still working fine on my end (using german ip too)
- what about your privacy settings in FF ?

![image](https://user-images.githubusercontent.com/10303732/98448710-644e5700-212e-11eb-98dd-e368b355e5e8.png)

**Comment by spodermenpls** [comment]
@mapx- Now it's working without redirects again on my end, too (with uMatrix active and additional Adguard lists, even). I've noticed that in the uBlock logger most of the 3rd-party resources are excluded with the @@ exemption from 'uBlock filters', which wasn't the case when I've experienced the problems (uBlock Annoyances, Easylist, Lowe were blocking them, hence triggering the anti.adblock mechanism. - uBlock filters were ticked then too, though).
All Firefox anti-tracking techniques are deactivated on my end, and can't interfere.
